test/queryRunner.test.ts

```diff
@@ -72,6 +72,10 @@ suite('Query Runner tests', () => {
         // ... Mock up the view and VSCode wrapper to handle requests to update view
         testStatusView.setup(x => x.executingQuery(TypeMoq.It.isAnyString()));
         testVscodeWrapper.setup( x => x.logToOutputChannel(TypeMoq.It.isAnyString()));
+        let testDoc: vscode.TextDocument = {
```

Comment:
Add tests

Reply:
Will add tests once the feature is fully in after the review.